src/components/LoanCollections/KivaClassicLoanCategorySelector.vue

```diff
@@ -35,7 +35,20 @@ export default {
 		}
 	},
 	methods: {
-		handleCategoryClick(categoryId) {
+		handleCategoryClick(category) {
+			const categoryId = category?.id ?? null;
+			const categoryShortName = category?.shortName ?? '';
+			// build event category from url, special case for homepage, clean page path otherwise
+			const eventContext = this.$route.fullPath === '/'
+				? 'homepage'
+				: this.$route.fullPath.replace(/\//g, '-').replace('-', '');
```

Comment:
I like this handling of changing the tracking category!

Reply:
Conceptually at least ;)